### DISCUSSION TITLE
Job is not done. ERROR

### DISCUSSION THREAD

**Discussion by mariuszmalek** [brefphp/bref#946] [Support] [answered]
I have a big problem with worker/queue. When I try to register in my application, then I have been blocked by error (image). After registration software sending a welcome e-mail (queue). Can anyone help me?

![image](https://user-images.githubusercontent.com/26696860/120894090-f95af500-c616-11eb-8a85-0caa38083cfd.png)

**Comment by mnapoli** [comment, marked as answer]
Are you publishing the jobs in the queue via Laravel?

Without more information it's impossible to help. It seems the job isn't really valid for Laravel.

**Comment by mnapoli** [comment]
Closing as duplicate of #948
